src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/ExtractCohortEngine.java

```diff
@@ -90,7 +91,8 @@ public ExtractCohortEngine(final String projectID,
                                final double vqsLodINDELThreshold,
                                final ProgressMeter progressMeter,
                                final ExtractCohort.QueryMode queryMode,
-                               final String filterSetName) {
+                               final String filterSetName,
+                               final String Avrofilepath) {
```

Comment:
put this next to the table parameter (`cohortTableName`) and use a similar name like `cohortAvroFileName`